### ISSUE TITLE
Vertex-Dark youtube.com search text, impossible to see.

### ISSUE THREAD

**Issue by fabiodellaria** [horst3180/vertex-theme#205]
Hi to all, GREAT theme, I really Love it! :)

The issue I'm experiencing is for the Vertex-Dark variant on the Youtube.com site.

As you can see from the attached image, the search text is so lightest to be impossible to see it. In the attached example I insert the text "sofia" in the youtube search box.

![vertex-dark-youtube](https://cloud.githubusercontent.com/assets/19651914/15832248/990d4d0e-2c21-11e6-9568-823d09519b29.png)

**Comment by fabiodellaria** [comment]
My wrong! :) I had only to install Vertex-Dark theme for Firefox! ;)

**Comment by jwarkentin** [comment]
The invisible text on YouTube and in some text boxes has been driving me crazy for a while. Installing the theme fixes that but unfortunately gets rid of the awesome default theme on Firefox Developer Edition. It would be nice if there was a solution to the text coloring issues that still allowed me to keep the default theme.

**Comment by FrankNitty** [comment]
@jwarkentin Agreed. Because I use Opera as my main browser and suffer from the invisible text especially on the bookmarks bar and also in youtube. I am not able to add the theme like in firefox.
![vertex theme invisible text](https://cloud.githubusercontent.com/assets/11983339/17607531/4fcc4ba2-5ff4-11e6-8aaa-1788ff006ac8.png)

**Comment by buha** [comment]
@fabiodellaria  How did you install it? 
I followed the instructions on Vertex theme github and it does not change anything.

**Comment by BigDaddyLinux** [comment]
@buha Thanks, I will give it a whirl
